client/state/ui/guided-tours/actions.js

```diff
@@ -49,3 +49,7 @@ function addSeenGuidedTour( getState, tourName, finished = false ) {
 		}
 	] );
 }
+
+export function resetGuidedTourHistory() {
```

Comment:
Hm. Even prior to this PR, we do have some places where we call GT _GuidedTour_, singular, which is a bit annoying. We should have _Guided Tours_ everywhere.